### ISSUE TITLE
Losing folder properties like svn:externals and svn:ignore

### ISSUE THREAD

**Issue by Zurd** [jasperlee108/svndumpfilterIN#14]
Hi, the tool works quite good, however we are losing svn properties like svn:externals and svn:ignore after loading the new dump.

The issue seems to be in the functions _extract_properties

**Comment by justinmassiot** [comment]
Hi @Zurd,
I've used svndumpfilterIN recently on big repositories and haven't notice any problem with properties. Can you give more details on what you do and is not as expected?

**Comment by Zurd** [comment]
Actually, after some testing & research, the SVN properties are still there so svndumpfilterIN is working fine. But some properties are missing, they are the ones that came from another branch.

Here's a more in-depth explanation: we have one big repository with many branch (projects) like so:
/branch1
/branch2
/branch3

We made a dump of this big repository and are using svndumpfilterIN to separate all the branch into different repository. Ultimately we want 1 repository for each branch. The first one we are doing right now, is branch3. However, at the start of this project, it used to be branch1 and at some point someone copied everything to branch3. I don't know how exactly since it's been done quite a long time ago by someone else.

All the SVN properties like svn:ignore, svn:externals, svn:executable, svn:specials, etc. from branch3 is being copied fine by svndumpfilterIN to the new repository, however all the folders and files that was copied from branch1 to branch3 are missing their properties.

As a solution, I was able to export all the properties from the big repository we have with:
svn proplist -v -R https://the-svn-repository/branch3/trunk/ > proplist.txt

Then I made a bash script that read the file proplist.txt containing all the SVN properties and output all the commands of "svn propset" to set the SVN properties on the new repository.
